<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleWidget</class>
 <widget class="QWidget" name="ConsoleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_co">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_co_co">
      <attribute name="icon">
       <iconset resource="Frostbite.qrc">
        <normaloff>:/frostbite/icons/console.png</normaloff>:/frostbite/icons/console.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="HistoryLineEdit" name="lineEdit_co_co">
         <property name="placeholderText">
          <string>Enter command to execute here...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_co_co">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textEdit_co_co">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_co_pb">
      <attribute name="icon">
       <iconset resource="Frostbite.qrc">
        <normaloff>:/frostbite/icons/punkbuster.png</normaloff>:/frostbite/icons/punkbuster.png</iconset>
      </attribute>
      <attribute name="title">
       <string>PunkBuster</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0" colspan="2">
        <widget class="QTextEdit" name="textEdit_co_pb">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="HistoryLineEdit" name="lineEdit_co_pb">
         <property name="placeholderText">
          <string>Enter command to execute here...</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_co_pb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryLineEdit</class>
   <extends>QLineEdit</extends>
   <header>HistoryLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Frostbite.qrc"/>
 </resources>
 <connections/>
</ui>
